.github/workflows/nodejs.yml

```
@@ -38,20 +38,96 @@ jobs:
 
     - name: Continuous integration
       run: |
-        npm run ci
+        npm run ci:20-
         npm run tsc:pub
 
     - name: Code Coverage
       uses: codecov/codecov-action@v1
```

actionlint (1.7.8)

the runner of "codecov/codecov-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)